Makefile

```diff
@@ -75,7 +75,9 @@ DOCKER_ENVS := \
 	-e PLATFORM \
 	-e DEFAULT_PRODUCT_LICENSE \
 	-e PRODUCT \
-	-e PACKAGER_NAME
+	-e PACKAGER_NAME \
+	-e OTEL_EXPORTER_OTLP_ENDPOINT \
+	-e OTEL_SERVICE_NAME
```

Comment:
> what if OTEL_EXPORTER_OTLP_ENDPOINT is something like localhost:12345

It will fail to connect and show error messages in the daemon/client logs, but wouldn't block anything, except maybe shutting down the tracer would error.

Basically, don't do that.

> do we need to have something to allow passing a socket (and bind-mount that into the container?)

I don't think there is any mechanism to connect over anything but TCP.